components/eam/bld/namelist_files/namelist_defaults_eam.xml

```diff
@@ -1627,10 +1628,10 @@ with se_tstep, dt_remap_factor, dt_tracer_factor set to -1
 <!-- Ice nucleation mods-->
 <use_hetfrz_classnuc phys="default">.true.</use_hetfrz_classnuc>
 <use_hetfrz_classnuc MMF_microphysics_scheme="sam1mom">.false.</use_hetfrz_classnuc>
-<use_preexisting_ice phys="default">.false.</use_preexisting_ice>
-<hist_hetfrz_classnuc phys="default">.false.</hist_hetfrz_classnuc>
+<use_preexisting_ice               >.false.</use_preexisting_ice>
```

Comment:
Is removing the selector going to affect other kind of physics? I see "cam4", "cam5" as other options...

Reply:
Actually, no other physics are used, and the changes I made here are consistent with changes that were made in E3SM/master (removing the selectors). cam4 and cam5 are relics from CESM, and are not even supported in E3SM and probably need to go away altogether. So I don't think there's any worry here.

Reply:
Great! Shall we merge then?